Update theme and styling

finnshort opened the pull request:
This PR gives the site a more modern feel and improves accessibility by using a colourblind accessible palette for the map icons and differentiating the icons by shape as well as colour. It requires upload the following two files to the BikeMaps/media/mapApp directory, which is not tracked by git.

![collision_teardrop](https://user-images.githubusercontent.com/3713996/171062114-1e5a967a-a227-4e39-a365-b35476b28721.png)

![nearmiss_teardrop](https://user-images.githubusercontent.com/3713996/171062101-46c667af-fe4e-4907-8668-8bc21c8fd4b3.png)

dgboss commented on mapApp/static/mapApp/js/icons.js:
```diff
@@ -20,18 +20,14 @@ L.ExtraMarkers.Icon.prototype.options.prefix = 'fa';
-    icon: "fa-bicycle",
-    markerColor: getColor("collision"),
-    iconColor: 'black',
+    innerHTML: '<img src="/media/mapApp/collision_teardrop.png" width=\'27px\' height=\'36px\' style=\'margin-top:5px\'>',
```
Can we include the two custom teardrop icons in source control somewhere?

finnshort replied:
I think the blogApp folder is excluded from git so that admin can upload images directly to the server, but I couldn't remember any reason why the mapApp folder would need to be. Just removed it from .gitignore but need to test further, ran into lots of little issues with images so want to make sure this works

dgboss reviewed (approved):
This is looking great! Just a couple thoughts for future consideration with respect to accessibility. 
1. These menu items are pretty low contrast
![image](https://user-images.githubusercontent.com/7094320/174940074-5da7f35f-d264-4816-b719-3213be6431fc.png)
2. Ideally all hyperlinks (like those in the legend) should be underlined

finnshort commented:
Thanks for the insight on the menu, I changed the text color to be the same as the legend/overlay which passes accessibility guidelines (https://webaim.org/resources/contrastchecker/). I added an underline on hover to the links and then took out the highlight because both seemed like too much.

dgboss commented:
Looks good!
